Description: Typo.
 .
 gmail-notify (1.6.1.1-4) experimental; urgency=medium
 .
   * Add sound and libnotify support
Author: Pedro Boliche <bolichep@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- gmail-notify-1.6.1.1.orig/notifier.py
+++ gmail-notify-1.6.1.1/notifier.py
@@ -168,7 +168,7 @@ class GmailNotify:
 		pixbuf = icon_theme.load_icon( icon_state , size, gtk.ICON_LOOKUP_FORCE_SVG)
 		scaled_buf = pixbuf.scale_simple(size,size,gtk.gdk.INTERP_BILINEAR)
 		self.tray.set_from_pixbuf(scaled_buf)
-		if state==2:
+		if state=='new':
 			self.tray.set_blinking(True)
 		else:
 			self.tray.set_blinking(False)	
